### ISSUE TITLE
Unable to deploy Vyper contracts

### ISSUE THREAD

**Issue by TeNNoX** [wighawag/hardhat-deploy#94]
I posted this issue [in the hardhat repo](https://github.com/nomiclabs/hardhat/issues/1365), but upon inspecting the source code I think it might also be an issue of this package...? (I have no idea which part is correct behaviour and which is not)

The issue is that compiled vyper contracts don't have buildInfo `.dbg.json` files:
![Screenshot from 2021-04-02 11-23-40](https://user-images.githubusercontent.com/2084639/113402783-eb35f180-93a5-11eb-95a3-baacfc2b776d.png)

But the DeploymentsManager needs them in:
- `getExtendedArtifact`
- which calls `utils.getExtendedArtifactFromFolder`
- which calls `Artifacts.getBuildInfo` (hardhat/internals/artifacts)
- which fails as there is no build info for the vyper contracts

**Workaround:** set `saveDeployments: false` for the network

Log:
```
Error: ERROR processing /home/manu/dev/uniwine-contracts/deploy/001_deploy.ts:
Error: cannot find artifact "Uniwine"
    at Object.getExtendedArtifact (/home/manu/dev/uniwine-contracts/node_modules/hardhat-deploy/src/DeploymentsManager.ts:227:17)
    at _deploy (/home/manu/dev/uniwine-contracts/node_modules/hardhat-deploy/src/helpers.ts:450:32)
    at _deployOne (/home/manu/dev/uniwine-contracts/node_modules/hardhat-deploy/src/helpers.ts:679:18)
    at DeploymentsManager.executeDeployScripts (/home/manu/dev/uniwine-contracts/node_modules/hardhat-deploy/src/DeploymentsManager.ts:997:19)
    at DeploymentsManager.runDeploy (/home/manu/dev/uniwine-contracts/node_modules/hardhat-deploy/src/DeploymentsManager.ts:832:5)
```

**Comment by wighawag** [comment]
Thanks for the report, I could make hardhat-deploy continue as indicated in https://github.com/nomiclabs/hardhat/issues/1365 but it would be preferable if hardhat-vyper add the necessary information.. 

I am not sure what vyper provide. from the [doc](https://vyper.readthedocs.io/en/latest/compiling-a-contract.html#vyper-json-output) I cannot find any reference to `metadata` that is used by hardhat-deploy to verify source code for solidity-based contract. So there might be some changes to do in hardhat-deploy to handle source verification for vyper contracts anyway

**Comment by kesar** [comment]
same issue here 👍🏻

**Comment by wighawag** [comment]
Hey @kesar @TeNNoX just a message that I hope to tackle this next week. thanks for the report

**Comment by wighawag** [comment]
fixed in v0.7.6 feel free to reopen if this does not work for you, thanks again for the report
